src/lang-promql/complete/hybrid.ts

```diff
@@ -319,7 +327,7 @@ export function analyzeCompletion(state: EditorState, node: SyntaxNode): Context
       result.push({ kind: ContextKind.Duration });
       break;
     case FunctionCallBody:
-      result.push({ kind: ContextKind.MetricName }, { kind: ContextKind.Function }, { kind: ContextKind.Aggregation });
+      result.push({ kind: ContextKind.MetricName, metricName: '' }, { kind: ContextKind.Function }, { kind: ContextKind.Aggregation });
```

Comment:
in that case you can remove the metricName.

Reply:
ah yeah ok no nethermind, it's coherent to do it like that (looking at the test, it would be weird to not having the metricName empty)